packages/flutter/lib/src/cupertino/picker.dart

```diff
@@ -313,6 +310,57 @@ class _CupertinoPickerState extends State<CupertinoPicker> {
   }
 }
 
+/// An picker default magnifier, grey rrect first introduced in iOS 14
+class CupertinoPickerDefaultMagnifier extends StatelessWidget {
+
+  /// Create a magnifier, it can be easily configured with the default style.
+  /// The default background is [CupertinoColors.tertiarySystemFill],
+  /// which also has default radius and margin on the left and right sides.
+  const CupertinoPickerDefaultMagnifier({
+    Key key,
+    this.background = CupertinoColors.tertiarySystemFill,
+    this.useLeftStyle = true,
```

Comment:
Could be more self-descriptive. Maybe `capLeftEdge`?

Reply:
It's a good name, I've been thinking about naming it for a long time.... Thank you! ~~ It's been updated now.